### PR TITLE
Initial changes to mirror to adoptium

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,19 +20,19 @@ jdk8u/... scripts
...
-  - "release"  = "master"(latest build tag) + "AdoptOpenJDK Patches"
+The Adoptium mirrors also have two vendor branches for AdoptOpenJDK to apply any extra patches needed:
+  - "dev"      = "master"(HEAD) + "Adoptium Patches"
+  - "release"  = "master"(latest build tag) + "Adoptium Patches"
 
-**Note For Developers:** Any AdoptOpenJDK Patches must be done on the "release" branch, they will be auto-merged nightly into "dev".
+**Note For Developers:** Any Adoptium Patches must be done on the "release" branch, they will be auto-merged nightly into "dev".
 
 The script merges the appropriate latest merged "master" branch code into both "dev" and "release", it also ensures all the
 "AdoptOpenJDK Patches" from the "release" branch are merged into the "dev" branch.
```
Comment (adamfarley) [inline README.md:38]: I think you missed one. :)

Comment (gdams) [reply README.md]: good spot thanks!

**Comment by adamfarley** [comment]
I think there are 4 references to AdoptOpenJDK (including one repo name) at the top of freebsdRepoSync.sh that needs changing.

**Comment by gdams** [comment]
> I think there are 4 references to AdoptOpenJDK (including one repo name) at the top of freebsdRepoSync.sh that needs changing.

@adamfarley I'm not changing the freebsd sync script yet because we don't actually need a freebsd mirror at adoptium right now

**Comment by adamfarley** [comment]
> 
> 
> > I think there are 4 references to AdoptOpenJDK (including one repo name) at the top of freebsdRepoSync.sh that needs changing.
> 
> @adamfarley I'm not changing the freebsd sync script yet because we don't actually need a freebsd mirror at adoptium right now

Ah, ok. I saw your other change to freebsdRepoSync.sh and I assumed there was something I didn't know.

Also, there's a few comments in mercurialToGit.sh and skaraMirror.sh that could be updated, but nothing more than "comments". All the functional stuff LGTM. 👍

**Pull request merged**